superset/databases/schemas.py

```diff
@@ -47,6 +47,11 @@
     SSHTunnelMissingCredentials,
 )
 from superset.constants import PASSWORD_MASK
+from superset.databases.types import (  # pylint:disable=unused-import
+    EncryptedDict,  # noqa: F401
+    EncryptedField,
+    EncryptedString,  # noqa: F401
+)
```

Comment:
If `EncryptedDict` and `EncryptedString` are not used, why are you importing them?

```suggestion
from superset.databases.types import EncryptedField
```

Reply:
They are being imported from db_engine_specs from that location. Rather than change all of the references in every spec, I thought it would be cleaner to still point to the Schemas, which is where they really should be located were it not for app_context issues.